### PR TITLE
Basic setup : Tailwind and Backend

### PR DESCRIPTION
**Pull request by Megha-Pathak**
Setup: 
- Tailwind 
- Backend using Mockbee 
- Favicon

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
```
Comment (suyashpradhan) [inline public/index.html:7]: Change description

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.App {
```
Comment (suyashpradhan) [inline src/App.css:1]: You don't need this right? If you are using tailwind

File: src/App.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import "./App.css";
+
+function App() {
+  return <div className="App"> hello world </div>;
```
Comment (suyashpradhan) [inline src/App.js:4]: Check previous comment

File: src/index.css [hunk 4]
```diff
@@ -0,0 +1,9 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@layer base {
+  html {
+    font-family: "Montserrat", sans-serif;
```
Comment (suyashpradhan) [inline src/index.css:7]: I would suggest using some different font, Montserrat looks big and messy if we have more content.
You can go for Inter, Rubik, Josefin Sans. Btw just a suggestion!

**Review by suyashpradhan — CHANGES_REQUESTED** [review]
The code looks good, Added a few suggestions

just wanted to know any particular reason for using mirage BE and not express, also would suggest using yup for validation

File: src/App.css (hunk 2 above)
Comment (Megha-Pathak) [reply src/App.css:1]: correct

**Pull request merged**